Python/RSSFeedReader/RSSFeedReader.py

```diff
@@ -1,9 +1,36 @@
 # RSSFeedReader
 
-# Set URL for Feed
+# Requests library used for get()
+# import requests
+import urllib.request
 
-# Request RSS Feed for set URL and collect output
+import xml.etree.ElementTree as ET
+
+# Set the RSS FeedURL
+URL = 'http://www.motortrend.com/widgetrss/motortrend-stories.xml'
 
-# print response output
+# Request RSS Feed for set URL and collect output
+try:
+  response = urllib.request.urlopen(URL)
+  status_code = response.getcode()
+  if not status_code == 200:
+    print(f'URL Request Open Error: {status_code}')
+    exit()
+  with response as RSSPage:
```

Comment:
Explain the with statement above.

Reply:
Turns out this is not needed at all.
I have removed it.